common/src/main/java/org/opensearch/ml/common/controller/MLRateLimiter.java

```diff
@@ -49,6 +50,14 @@ public static MLRateLimiter parse(XContentParser parser) throws IOException {
             switch (fieldName) {
                 case LIMIT_FIELD:
                     limit = parser.text();
+                    try {
+                        int limitNumber = Integer.parseInt(limit);
```

Comment:
What if the number is zero?

Reply:
Zero will simply build a rate limiter that refuse all requests, so essentially this is a legal case.

Reply:
Makes sense. Do we support fraction, like 1.5?

Reply:
Nice catch, I just test, fraction works so I change int into double